### ISSUE TITLE
HongKong_staff_data _raw

### ISSUE THREAD

**Issue by katiesw** [dragondave/cameron_diversity#5]
- [ ] Hong Kong uni staff 2015_16.pdf
- [ ] Hong Kong staff 2016_2017.pdf

**Comment by dragondave** [comment]
Have you shared these files with me in some way?

**Comment by cameronneylon** [comment]
No, it looks like we haven't. Apologies for radio silence, its been a mad few weeks with too much going on. The files are at: https://drive.google.com/drive/folders/1YQ8MMenw9YrxYBTfwLiHxscRw0vDNfDw?usp=sharing 

...does the general approach of noting the key tables in an issue work? We're just finalising the overall data format we think we'll need.

**Comment by dragondave** [comment]
It is useful to note the tables you're interested in, saves additional work!
I assume the new data'll be in the drive folder?

**Comment by cameronneylon** [comment]
Yes, the Hong Kong data should hopefully be in the drive folder.


On Tue, Jun 25, 2019 at 9:30 PM, dragondave <notifications@github.com> wrote:
It is useful to note the tables you're interested in, saves additional work!
I assume the new data'll be in the drive folder?

**Comment by dragondave** [comment]
I'm not seeing it in
https://drive.google.com/drive/folders/1rJ_LXsgKn3owCc4KXECHTLfaWZ2uBMWo --
could you confirm where it is?

**Comment by cameronneylon** [comment]
Sorry, it was in another folder. Added it to the link you just shared now.

On Thu, Jun 27, 2019 at 11:06 PM, dragondave <notifications@github.com> wrote:
I'm not seeing it in
https://drive.google.com/drive/folders/1rJ_LXsgKn3owCc4KXECHTLfaWZ2uBMWo --
could you confirm where it is?
